### ISSUE TITLE
Correction du INBALANCE 

### ISSUE THREAD

**Issue by pleisy** [pleisy/ML_EMS#4]
A faire seulement sur le TRAINING set

Mais pas sur la validation ou le test SET pour garder la distribution originelle
